apple/utils.bzl

```diff
@@ -20,41 +20,38 @@ XCRUNWRAPPER_LABEL = "@bazel_tools//tools/objc:xcrunwrapper"
 DARWIN_EXECUTION_REQUIREMENTS = {"requires-darwin": ""}
 """Standard execution requirements to force building on Mac."""
 
+_EXECUTION_REQUIREMENTS = {
+    "no_cache": "no-cache",
+    "no_sandbox": "no-sandbox",
+    "no_remote": "no-remote",
+}
+
+def _add_execution_requirements(kw):
```

Comment:
this method would modify the given kw object, which might affect other users upstream if they were sharing the kw dictionary.

make a copy of kw with dict(kw), and perhaps rename to `args_with_execution_requirements` as it wouldn't be adding stuff to the original kw

Reply:
good point